collada/scene.py

```diff
@@ -314,7 +314,7 @@ class Node(SceneNode):
     Contains the list of transformations effecting the node as well as any children.
     """
 
-    def __init__(self, id, children=None, transforms=None, xmlnode=None):
+    def __init__(self, id, children=None, transforms=None, xmlnode=None, name=None):
```

Comment:
Could you update the unit test? Probably here:
https://github.com/pycollada/pycollada/blob/master/collada/tests/test_scene.py#L166